flair/data.py

```diff
@@ -588,7 +588,12 @@ def __copy__(self):
         return s
 
     def __str__(self) -> str:
-        return 'Sentence: "{}" - {} Tokens'.format(' '.join([t.text for t in self.tokens]), len(self))
+
+        if len(self.labels) == 0:
```

Comment:
To simplify the check you could also do:
```
if self.labels:
  return f'Sentence: "{self.to_tokenized_string()}" - {len(self)} Tokens - Labels: {self.labels}'
else:
  return f'Sentence: "{self.to_tokenized_string()}" - {len(self)} Tokens'
```

Reply:
Thanks - jusu added this!